tests/Backend/Exasol/Workspace/LegacyWorkspaceLoadTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Keboola\Test\Backend\Exasol\Workspace;
```

Comment:
tohle by se melo asi uz povolit

Reply:
jak povolit? testDottedDestination nikdy nepovolíš, protože nepojede na exasolu. Ostatní je povolené v tvojém PR, který přidal ty alter věci.

Reply:
jo, uz to vidim. Diky